src/modules/renderingPokemon.js

```diff
@@ -0,0 +1,64 @@
+import postLikes from './postLikes.js';
+import { countLikes } from './likeCounter.js';
+import createPokeImage from './createImg.js';
+import { Popup, addPopup } from './popup.js';
+
+export default function renderPokemon(pokeData) {
+  const allPokemonContainer = document.getElementById('cards-container');
+  // create card
```

Comment:
- [ ] Great job, :clap: but I think it will be better if you can use ES6 arrow syntax for functions instead of a  function declaration for more information [JavaScript Function Expressions](https://www.w3docs.com/learn-javascript/function-expressions.html). Kindly make this for all your code